client/.eslintrc.yml

```diff
@@ -0,0 +1,28 @@
+---
+parser: babel-eslint
+
+extends: eslint-config-shakacode
+
+plugins:
+  - react
+
+globals:
+  __DEBUG_SERVER_ERRORS__: true
+  __SERVER_ERRORS__: true
+
+env:
+  browser: true
+  node: true
+  mocha: true
+
+rules:
+  ### Imports
+  # Can't do next one because of alias in webpack
```

Comment:
this can be resolved via https://www.npmjs.com/package/eslint-import-resolver-webpack but what are we aliasing in the hello world example?